docs/1-general-configuration.md

```diff
@@ -206,7 +206,7 @@ ActiveAdmin.setup do |config|
   config.namespace :admin do |admin|
     admin.build_menu :utility_navigation do |menu|
       menu.add label: "ActiveAdmin.info", url: "https://www.activeadmin.info",
-                                          html_options: { target: :blank }
+                                          html_options: { target: :_blank }
```

Comment:
@javierjulio isn't it better to have a string now that this symbol is changing?

Reply:
Hi @tagliala, I've changed it to use strings instead of symbols

Reply:
No, it won't matter because it will still covert that to a string. A symbol with a leading underscore is valid. I would have left the format as is but doesn't matter if its a string or not.